py-polars/polars/io/database.py

```diff
@@ -249,6 +250,7 @@ def _from_arrow(
         batch_size: int | None,
         iter_batches: bool,
         schema_overrides: SchemaDict | None,
+        **kwargs: Any,
```

Comment:
Why do we need to introduce an unused `**kwargs` here? I see it's on `read_database` as well, but I am really opposed to unused `**kwargs` as it allows you to input non-existing parameters (e.g. misspell some existing param / conjure up some parameter you think should work) without getting an error.

Reply:
I can tweak if you like. The `_from_arrow` and `_from_rows` signatures need to match; as this param isn't relevant for `_from_arrow` the **kwargs is just a way to deal with it without implying it is going to get used. As it's a private method there's no external API exposure.

The `read_database` kwargs param is unrelated, and only exists due to the deprecation passthrough for `read_database_uri`; as soon as we make the break between the two we can ditch the `read_database` kwargs (maybe next version?)